test/externalTests/gnosis.sh

```diff
@@ -87,6 +87,10 @@ function gnosis_safe_test
     sed -i "s|it\(('can only be called from Safe itself'\)|it.skip\1|g" test/libraries/Migration.spec.ts
     sed -i "s|it\(('should enforce delegatecall to MultiSend'\)|it.skip\1|g" test/libraries/MultiSend.spec.ts
 
+    jq '.overrides."@ethersproject/abstract-provider"="5.6.0" |
```

Comment:
Please add a comment explaining why we are doing this and when we can remove it.

Reply:
I would be fine with adding a comment as to why we're doing it, but realistically, is there any point in opening yet another issue that will not be addressed?

Reply:
It does not look like an upstream issue to me so yeah, you should not report it. I think we only have this problem because we pinned version for `ethers`.

But please do add the comment. Each hack should be commented to encourage people to try to remove it.

Reply:
Actually, maybe you could try to remove the `ethers` hack and some of the others we have here? The original problem that made us pin `ethers` at 5.6.1 has been resolved several versions ago so we should have been able to remove it but there were some problems with it when @matheusaaguiar tried it last time. See #13051 and https://github.com/ethers-io/ethers.js/discussions/2849#discussioncomment-2812920 for some context.

If our current problem is caused by the old workarounds, removing them might help. Not having to pin version should also make it less likely that we'll hit another incompatibility soon.

Reply:
Unfortunately, removing any of the old hacks causes failures, so it looks like they'll have to stay for the time being.

Reply:
But what kind of failures? Maybe it's something that has an easy solution?

Reply:
Unchanged, i.e. the same issue with a mass of reverts expectations not being met (issue references in `TODO`).